service/create/cloudconfig.go

```diff
@@ -0,0 +1,58 @@
+package create
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/base64"
+	"io/ioutil"
+	"text/template"
+
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+type CloudConfig struct {
```

Comment:
I think this should be part of this `clustercloudconfig` repo idea I talked about above.